examples/blog-starter/components/syntax-highlight.css

```diff
@@ -1,152 +0,0 @@
-code[class*='language-'],
```

Comment:
Why was this file deleted? Was it not referenced? Maybe it needs to be, can we add to _app?